packages/playwright/src/generators/init-deployment/utils/update-azdevops-build.ts

```diff
@@ -18,11 +18,8 @@ export function updateAzureDevopsStages(tree: Tree) {
 
         if (scriptIndex && stages?.stages[0]?.jobs[0]?.steps[scriptIndex]) {
             stages.stages[0].jobs[0].steps[scriptIndex] = {
-                script:
-                    'npm ci\n' +
-                    'git config --global user.email "pipelines@test.dev"\n' +
-                    'git config --global user.name "Amido Pipelines"\n' +
-                    'npx playwright install --with-deps',
+                // eslint-disable-next-line no-useless-concat
+                script: 'npm ci\n' + 'npx playwright install --with-deps',
```

Comment:
The init-deployment will have already defined this step with the 'git config' by pulling values from the nx.json file. So we're essentially going to delete that when we run the playwright init deployment generator.

Ideally, could we get the existing script for this step, then append 'npx playwright install --with-deps' to it?